src/components/menu/menu.spec.ts

```diff
@@ -320,6 +339,76 @@ describe('MenuComponent', () => {
         expect(component.trigger.menu._isFocused$.value).toBeFalsy();
         expect(component.subMenuTrigger.menu._isHovering$.value).toBeFalsy();
         expect(component.subMenuTrigger.menu._isFocused$.value).toBeFalsy();
+    });
+});
+
+@Component({
+    selector: 'app-menu-ngfor-test',
+    template: `
+        <div class="btn-group" *ngIf="showTrigger">
+            <button
+                type="button"
+                class="btn button-secondary dropdown-toggle"
+                [uxMenuTriggerFor]="menu">
+                Actions
+            </button>
+        </div>
+
+        <ux-menu #menu>
+            <button type="button" uxMenuItem>
+                <span class="dropdown-menu"></span>
+                <span class="dropdown-menu-text">Export</span>
+            </button>
+
+            <button type="button" uxMenuItem>
+                <span class="dropdown-menu"></span>
+                <span class="dropdown-menu-text">Annotate</span>
+            </button>
+
+            <button type="button" uxMenuItem>
+                <span class="dropdown-menu"></span>
+                <span class="dropdown-menu-text">Save List</span>
+            </button>
+        </ux-menu>
+    `
+})
+export class MenuNgForTestComponent {
```

Comment:
Just need to change this and related strings to e.g. `MenuTriggerDestroyTestComponent` since it no longer does anything with ngFor.